src/main/java/seedu/address/ui/StudentCard.java

```diff
@@ -52,7 +60,40 @@ public StudentCard(Student student, int displayedIndex) {
         email.setText(student.getEmail().value);
         student.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
-                .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+                .forEach(tag -> tags.getChildren().add(createTag(tag)));
+    }
+
+    /**
+     * Creates a {@code Label} with the given {@code Tag} details.
+     */
+    public Node createTag(Tag tag) {
```

Comment:
Is it possible to change `Node` to `Label`? 
```suggestion
     * Creates a {@code Label} with the given {@code Tag} details.
     */
    public Label createTag(Tag tag) {
```
Or alternatively change the `Label` in JavaDoc to `Node`?

Reply:
I think it would be best to define the return type as `Label` since we do in fact return a `Label`.